app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/PrivateBrowsingDescriptionViewHolder.kt

```diff
@@ -4,36 +4,122 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.style.TextDirection
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.LifecycleOwner
+import mozilla.components.ui.colors.PhotonColors
 import org.mozilla.fenix.R
-import org.mozilla.fenix.databinding.PrivateBrowsingDescriptionBinding
-import org.mozilla.fenix.ext.addUnderline
+import org.mozilla.fenix.compose.ComposeViewHolder
 import org.mozilla.fenix.home.sessioncontrol.TabSessionInteractor
+import org.mozilla.fenix.theme.FirefoxTheme
 
+/**
+ * View holder for a private browsing description.
+ *
+ * @param composeView [ComposeView] which will be populated with Jetpack Compose UI content.
+ * @param viewLifecycleOwner [LifecycleOwner] life cycle owner for the view.
+ * @param interactor [TabSessionInteractor] which will have delegated to all user interactions.
+ */
 class PrivateBrowsingDescriptionViewHolder(
-    view: View,
-    private val interactor: TabSessionInteractor
-) : RecyclerView.ViewHolder(view) {
+    composeView: ComposeView,
+    viewLifecycleOwner: LifecycleOwner,
+    val interactor: TabSessionInteractor,
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
 
     init {
-        val resources = view.resources
-        val appName = resources.getString(R.string.app_name)
-        val binding = PrivateBrowsingDescriptionBinding.bind(view)
-        binding.privateSessionDescription.text = resources.getString(
-            R.string.private_browsing_placeholder_description_2, appName
+        val horizontalPadding =
+            composeView.resources.getDimensionPixelSize(R.dimen.home_item_horizontal_margin)
+        composeView.setPadding(horizontalPadding, 0, horizontalPadding, 0)
+    }
+
+    @Composable
+    override fun Content() {
+        PrivateBrowsingDescription(
+            onLearnMoreClick = interactor::onPrivateBrowsingLearnMoreClicked
         )
-        with(binding.privateSessionCommonMyths) {
-            movementMethod = LinkMovementMethod.getInstance()
-            addUnderline()
-            setOnClickListener {
-                interactor.onPrivateBrowsingLearnMoreClicked()
-            }
-        }
     }
 
     companion object {
-        const val LAYOUT_ID = R.layout.private_browsing_description
+        val LAYOUT_ID = View.generateViewId()
+    }
+}
+
+/**
+ * Private browsing mode description.
+ *
+ * @param onLearnMoreClick Invoked when the user clicks on the learn more link.
+ */
+@Composable
+fun PrivateBrowsingDescription(
+    onLearnMoreClick: () -> Unit,
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+    val color = PhotonColors.LightGrey05 // Equivalent to fx_mobile_private_text_color_primary.
```

Comment:
will we long term need to have a private color palette in FirefoxTheme?

Reply:
The plan is to add Private Theme to the FirefoxTheme. I have thought about it but a solution has not appeared to me yet.

Reply:
At a time when it wasn't clear what colors will the app use or whether we'll roll our own theme or stick with `MaterialTheme` I started work on a `FenixTheme` which had support for light/dark and private mode also - https://github.com/mozilla-mobile/fenix/pull/21193/files